scRNA-seq/03-dimension_reduction_scRNA-seq.Rmd

```diff
@@ -20,63 +20,62 @@ downstream analyses.
 
 ## Set Up 
 
-For these analyses, we will need Seurat as well as the dplyr pipe. 
-
 ```{r}
-library(Seurat)
+# Load libraries
+library(ggplot2)
 
 # Magrittr pipe
 `%>%` <- dplyr::`%>%`
 ```
 
-We need to reload the Seurat object with the data we set up and normalized 
-previously. 
-
-```{r Path to Seurat}
-# Load in Seurat object
-seurat_norm <- readRDS(file.path("data", "seurat_object.RDS"))
+For this module, we are going to use the dataset that we processed part of in 
+`02-tag-based_pre-processing_scRNA-seq.md`
+We have already filtered and `scran` normalized this data for you (following 
+the same kinds of steps we did in `01-normalizing_scRNA-seq.Rmd`).
+We also randomly subsampled 500 cells, and are only using the top 95% highest 
+average expression genes.
+
+```{r Load in Tabula Muris Data}
+# Read in gene matrix
+tab_mur_data <- readr::read_tsv(file.path("data", "scran_norm_tab_mur.tsv"), 
+                                progress = FALSE) 
+
+# Read in associated metadata
+tab_mur_meta <- readr::read_tsv(file.path("data", "tab_mur_metadata.tsv"), 
+                                progress = FALSE)
 ```
 
-Remember that previously, we filtered our dataset, so as we import our metadata
-we need to filter it so that it only contains the cells we are working with, 
-in the order we are working with them in the data. 
+## Principal Components Analysis
 
-```{r Filter metadata}
-# Get list of cells after we previously filtered the data
-cells <- colnames(seurat_norm@raw.data)
+Just like we did with our other scRNA-seq data, we will also run PCA with our 
+single-cell data. 
```

Comment:
`run PCA on this dataset`?

Reply:
Yeah, this sentence is weird now that I look at it. I will change it.